proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -436,3 +436,12 @@ message GetDLQTasksResponse {
     // there are more results.
     bytes next_page_token = 2;
 }
+
+message PurgeDLQTasksRequest {
+  temporal.server.api.common.v1.HistoryDLQKey dlq_key = 1;
+  temporal.server.api.common.v1. HistoryDLQTaskMetadata inclusive_max_task_metadata = 2;
+}
+
+message PurgeDLQTasksResponse {
+  string job_id = 1;
```

Comment:
I'm not going to comment on the format of this because it should be opaque. It happens to be worfklowID/runID, but that's an implementation detail. A subsequent PR will add DescribeDLQJob and CancelDLQJob.

Reply:
Changed to be `bytes`. It's not truly opaque for now because the describe and cancel APIs aren't implemented yet so the tests still have to parse the token.